server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -989,6 +990,8 @@ public IndexSettings(final IndexMetadata indexMetadata, final Settings nodeSetti
         assignedOnRemoteNode = RemoteStoreNodeAttribute.isRemoteDataAttributePresent(this.getNodeSettings());
         remoteStorePathStrategy = RemoteStoreUtils.determineRemoteStorePathStrategy(indexMetadata);
 
+        isTranslogMetadataEnabled = RemoteStoreUtils.determineTranslogMetadataEnabled(indexMetadata);
```

Comment:
`determineTranslogMetadataEnabled ` -> `isTranslogMetadataEnabled `

Reply:
Let's keep this as `determineTranslogMetadataEnabled` as we use indexMetadata to determine this flag's value. Also this might confuse with getter method of this instance field.